### PR TITLE
WebTop-5 manual en

### PR DESCRIPTION
**Pull request by Ilvo**
webtop 5  manual integration

### CODE REVIEW

File: administrator-manual/en/webtop5.rst [hunk 1]
```diff
@@ -331,6 +362,58 @@ A precompiled wizard will open:
 
 Proceed with the Next button until the Wizard is complete.
 
+Use the personal Cloud to send and receive documents
+====================================================
+
+Using Cloud module allows you to send and receive documents throug web links.
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: remove the "Using" word.

File: administrator-manual/en/webtop5.rst [hunk 2]
```diff
@@ -331,6 +362,58 @@ A precompiled wizard will open:
...
+
+.. note::
+
+   The server must be reachable in http on port 80
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: Add a empty line here before the next title.

File: administrator-manual/en/webtop5.rst [hunk 3]
```diff
@@ -331,6 +362,58 @@ A precompiled wizard will open:
...
+Downloading the file, generates a notification to the sender:
+
+.. image:: _static/webtop-oc_cloud6.png
+Request for a document
+----------------------
+To create the request, insert the subject of the email than select the button at the top right:
+
+.. image:: _static/webtop-doc_cloud7.png
+
+Follow the wizard. You can set both an expiration date and a password. The link will be automatically inserted in the email:
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: "in the email" -> "into the message"

File: administrator-manual/en/webtop5.rst [hunk 4]
```diff
@@ -412,6 +495,27 @@ that can be used within the parameterized fields within the domain mailcard edit
 
 In this way it is possible to create a single mailcard that will be automatically customized for every user who does not use his own mailcard.
 
+Configure multiple mailcards for a single user
+==============================================
+
+It is possible to configure multiple mailcards (html signatures) for each individual user.
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: "html" is an abbreviation, it should be uppercase: "HTML"

File: administrator-manual/en/webtop5.rst [hunk 5]
```diff
@@ -511,6 +615,18 @@ Proceed as follows:
...
 5. the next login will show the new logo on the login page
 
+Change the public URL
+=====================
+
+A special prop has been added to modify the public URL.
+
+If you want to change URL from this: ``http://server.domain.local/webtop`` to: ``http://mail.publicdomain.com/webtop``
+
+execute these commands:
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: change the syntax, use the `::` operator

File: administrator-manual/en/webtop5.rst [hunk 6]
```diff
@@ -249,6 +248,38 @@ the check box is named :guilabel:`Show quick preview on message row`.
...
+
+ There are three archiving criteria in :menuselection:`Settings -> Mail -> Archiving`
+
+* **Single folder:** a single root for all archived emails
+* **Per year:** a root for each year
+* **By year / month:** a root for each year and month
+
+.. image:: _static/webtop-archive_archive2.png
+
+To maintain the original structure of the folders is possible to activate **"Keep folder structure"**
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: Enclose "Keep folder structure" inside the ``guilabel`` tag.

File: administrator-manual/en/webtop5.rst (hunk 1 above)
Comment (Ilvo) [reply administrator-manual/en/webtop5.rst]: done

File: administrator-manual/en/webtop5.rst (hunk 2 above)
Comment (Ilvo) [reply administrator-manual/en/webtop5.rst]: done

File: administrator-manual/en/webtop5.rst (hunk 3 above)
Comment (Ilvo) [reply administrator-manual/en/webtop5.rst]: done

File: administrator-manual/en/webtop5.rst (hunk 4 above)
Comment (Ilvo) [reply administrator-manual/en/webtop5.rst]: done

File: administrator-manual/en/webtop5.rst (hunk 6 above)
Comment (Ilvo) [reply administrator-manual/en/webtop5.rst]: done

File: administrator-manual/en/webtop5.rst (hunk 5 above)
Comment (Ilvo) [reply administrator-manual/en/webtop5.rst]: done

File: administrator-manual/en/webtop5.rst [hunk 7]
```diff
@@ -511,6 +617,18 @@ Proceed as follows:
...
 5. the next login will show the new logo on the login page
 
+Change the public URL
+=====================
+
+A special prop has been added to modify the public URL.
+
+If you want to change URL from this: ``http://server.domain.local/webtop`` to: ``http://mail.publicdomain.com/webtop``
+
+execute these commands::
```
Comment (gsanchietti) [inline administrator-manual/en/webtop5.rst]: Still, this is an invalid syntax. :)

**Comment by gsanchietti** [comment]
Manually merged.

**Pull request closed without merging**